.changeset/curvy-moles-punch.md

```diff
@@ -5,4 +5,4 @@
 Rework `typographyOptions`
 
 - Replace `typographyOptions` with `createTypographyOptions()` to enable using the theme's breakpoints for media queries
-- Add new styles for `button` variant
```

Comment:
This was removed again in the previous PR but I forgot to remove it from the changeset